ember.js/2017-12/december-08.md

```diff
@@ -0,0 +1,9 @@
+# Meeting: Ember Core Team 2017/12/08
+
+## Agenda
+
+* Skipped due to Core Team F2F
```

Comment:
Do there happen to be minutes from the F2F?

Reply:
There's an advent calendar of RFCs :P